packages/common/sagas/entities/team.js

```diff
@@ -128,11 +138,38 @@ export function* createTeamSaga({ payload: { title, icon } }) {
     const { data } = yield call(postCreateTeam, { title, icon });
```

Comment:
Please, rename the data var to the team var.

Reply:
Done